src/components/CInput/index.tsx

```diff
@@ -26,6 +27,9 @@ interface CInputProps {
   enterKeyHint?: 'search' | 'done' | 'enter' | 'go' | 'next' | 'previous' | 'send';
   onKeyPress?: React.KeyboardEventHandler<HTMLInputElement>;
   maxLength?: number;
+  copy?: boolean;
+  hideCharacter?: boolean;
+  copyOnClick?: () => void;
 }
```

Comment:
Remove these props, make a new component that uses CInput inside. Use chat component in Setting page.